build/async.js

```diff
@@ -0,0 +1,1225 @@
+'use strict';
```

Comment:
Ah nevermind, I get it -- we publish the `build/` folder.

My comments remain - I'd rather have a `dist/` folder under `build/` with the built `async.js` and `async.min.js` files.

I'm not sure if I want `es/` to be published to npm. I think I would prefer it to be published in a second repo (`async-es` or similar)